solr/test-framework/src/java/org/apache/solr/client/solrj/apache/SolrPortAwareCookieSpecFactory.java

```diff
@@ -31,6 +31,7 @@
 import org.apache.http.protocol.HttpContext;
 
 @SuppressWarnings("deprecation")
+@Deprecated
```

Comment:
[Q] Why is this deprecated in this PR?  Is it just a part of the larger "apache client" deprecation that was missed until now?

Reply:
Yes; notice the package name.
This really underscores to me why our different implementations ought to get their very own packages!